extensions/wikia/Email/EmailController.class.php

```diff
@@ -189,22 +194,45 @@ protected function getSubject() {
 	 * @return string
 	 */
 	protected function getBody() {
-		$moduleBody = $this->app->renderView(
-			get_class( $this ),
-			'body',
-			$this->request->getParams()
-		);
+		$css = file_get_contents( __DIR__ . '/styles/main.css' );
 
 		$html = $this->app->renderView(
 			get_class( $this ),
 			'main',
 			[
-				'content' => $moduleBody,
+				'content' => $this->getContent(),
 				'footerMessages' => $this->getFooterMessages()
-
 			]
 		);
 
+		$html = $this->inlineStyles( $html, $css );
+
+		return $html;
+	}
+
+	/**
+	 * Renders the content unique to each email. Should be overridden in child classes.
+	 */
+	protected function getContent() {
```

Comment:
at one point EmailController was abstract, but we had to change that. It looks like we might not need that change anymore so I'll see if I can make it abstract again.